### DISCUSSION TITLE
snap app icon went missing after upgrading pop!_os from 20.04 to 20.10

### DISCUSSION THREAD

**Discussion by SteveDevOps** [standardnotes/forum#2477] [General]
**Describe the bug**
snap app Icon was missing in pop!_os 20.10 -- when searching for app in application launcher and icon pinned to favorites bar.

I am running pop on vmware fusion vm on macbook pro catalina
This problem occurred after upgrading to pop 20.10 from 20.04
removal and reinstall of snap didn't fix.

found issue in .desktop file... after upgrading OS.

was formerly:
**Icon=/snap/standard-notes/11/meta/gui/icon.png**

found corect path / fixed to be:
**Icon=/snap/standard-notes/current/meta/gui/icon.png**

currently running:

snap 2.48.2
snapd 2.48.2
series 16
pop 20.10
kernel 5.8.0-7630-generic

zsh 5.8 (x86_64-ubuntu-linux-gnu)

**Comment by SteveDevOps** [comment]
There was an issue specific to zsh related to /snap/bin not coming over to path environment var -- but mine has it. The issue I encountered might in some way be related to this one as well.. but not sure. My manual fixes have remained in play and icons restored. I also discovered one other snap that I had with a missing icon after posting this and I fixed the same way.

https://forum.snapcraft.io/t/missing-icons-for-all-apps-installed-using-snap/14783

**Comment by arielsvg** [comment]
Hey @SteveDevOps, thanks for the report 🙂 do you believe there is something we as app-makers can do to help with this situation? Or is this more of an issue with pop!_os?

**Comment by SteveDevOps** [comment]
I'm not sure if this is directly related to os upgrade or snap upgrade -- I think it's definitely one of the two because I had discovered an additional snap with same problem

**Comment by Poecilia** [comment]
I have this issue in Pop_OS too, both with snap and direct download from the website. The desktop icon is missing.
